src/Mod/PartDesign/App/FeatureHelix.cpp

```diff
@@ -77,21 +77,22 @@ Helix::Helix()
 {
     addSubType = FeatureAddSub::Additive;
 
-    ADD_PROPERTY_TYPE(Base, (Base::Vector3d(0.0, 0.0, 0.0)), "Helix", App::Prop_ReadOnly, "Base");
-    ADD_PROPERTY_TYPE(Axis, (Base::Vector3d(0.0, 1.0, 0.0)), "Helix", App::Prop_ReadOnly, "Axis");
-    ADD_PROPERTY_TYPE(Pitch, (10.), "Helix", App::Prop_None, "Pitch");
-    ADD_PROPERTY_TYPE(Height, (30.0), "Helix", App::Prop_None, "Height");
-    ADD_PROPERTY_TYPE(Turns, (3.0), "Helix", App::Prop_None, "Turns");
+    const char* group = "Helix";
+    ADD_PROPERTY_TYPE(Base, (Base::Vector3d(0.0, 0.0, 0.0)), group, App::Prop_ReadOnly, "The center point of the helix' start.");
+    ADD_PROPERTY_TYPE(Axis, (Base::Vector3d(0.0, 1.0, 0.0)), group, App::Prop_ReadOnly, "The helix' direction.");
+    ADD_PROPERTY_TYPE(Pitch, (10.), group, App::Prop_None, "The axial distance between two turns.");
+    ADD_PROPERTY_TYPE(Height, (30.0), group, App::Prop_None, "The total length of the helix (not accounting for the extent of the profile)");
+    ADD_PROPERTY_TYPE(Turns, (3.0), group, App::Prop_None, "The number of turns in the helix (does not need to be a whole number).");
     Turns.setConstraints(&floatTurns);
-    ADD_PROPERTY_TYPE(LeftHanded, (long(0)), "Helix", App::Prop_None, "LeftHanded");
-    ADD_PROPERTY_TYPE(Reversed, (long(0)), "Helix", App::Prop_None, "Reversed");
-    ADD_PROPERTY_TYPE(Angle, (0.0), "Helix", App::Prop_None, "Angle");
-    ADD_PROPERTY_TYPE(Growth, (0.0), "Helix", App::Prop_None, "Growth");
+    ADD_PROPERTY_TYPE(LeftHanded, (long(0)), group, App::Prop_None, "Determins whether the turning direction of helix follows the \"left-hand rule\", i.e. counter-clockwise when moving along its axis.");
+    ADD_PROPERTY_TYPE(Reversed, (long(0)), group, App::Prop_None, "Determins whether the helix points in the opposite direction of the reference axis.");
+    ADD_PROPERTY_TYPE(Angle, (0.0), group, App::Prop_None, "If non-zero, the profile's path wraps around a cone with this angle, instead of a cylinder. Positive values make the radius grow, nevatige shrink.");
+    ADD_PROPERTY_TYPE(Growth, (0.0), group, App::Prop_None, "The growth of the helix' radius per turn. Non-zero values turn the helix into a conical spiral.");
     Angle.setConstraints(&floatAngle);
-    ADD_PROPERTY_TYPE(ReferenceAxis, (0), "Helix", App::Prop_None, "Reference axis of revolution");
-    ADD_PROPERTY_TYPE(Mode, (long(0)), "Helix", App::Prop_None, "Helix input mode");
-    ADD_PROPERTY_TYPE(Outside, (long(0)), "Helix", App::Prop_None, "Outside");
-    ADD_PROPERTY_TYPE(HasBeenEdited, (long(0)), "Helix", App::Prop_None, "HasBeenEdited");
+    ADD_PROPERTY_TYPE(ReferenceAxis, (0), group, App::Prop_None, "The reference axis of the helix.");
+    ADD_PROPERTY_TYPE(Mode, (long(0)), group, App::Prop_None, "The helix input mode.");
```

Comment:
I would say it is more the definition mode than the input, but I am also no native speaker.

Reply:
Indeed, that not very helpful. In fact I only changed the previous doc string to become a complete sentences.

Better would be: "The helix input mode specifies which properties are set by the user. Dependent properties are then calculated."